tests/v2/codecoverage/scripts/build_docker_images.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -e
 
+cd $(dirname $0)/../../../../
+source $(dirname $0)/scripts/export-config
+cd -
```

Comment:
We don't need to call cd here
```suggestion
source $(dirname $0)/../../../../scripts/export-config
```